[IMP] accounting: Update customer invoices page for 18.0 (#11813)
odoo/documentation: content/applications/finance/accounting/customer_invoices.rst

  upon confirmation.
- :guilabel:`Due Date` or :doc:`payment terms <customer_invoices/payment_terms>`: To specify when
  the customer has to pay the invoice.
- :guilabel:`Journal`: Automatically set and can be changed if needed.
- :doc:`Currency <get_started/multi_currency>`. If the invoice's currency differs from the
  company's currency, the currency exchange rate is automatically displayed.
- :guilabel:`Product`: Click :guilabel:`Add a line` to add a product.
- :guilabel:`Quantity`
- :guilabel:`Price`
- :doc:`Taxes <taxes>` (if applicable)

To access the product catalog, click :guilabel:`Add a line`, and :guilabel:`Catalog`. Then, update

> quantities and :guilabel:`Add` or :guilabel:`Remove` products from the invoice line.

Author: You're right, thanks for the clarification!

  upon confirmation.
- :guilabel:`Due Date` or :doc:`payment terms <customer_invoices/payment_terms>`: To specify when
  the customer has to pay the invoice.
- :guilabel:`Journal`: Automatically set and can be changed if needed.
- :doc:`Currency <get_started/multi_currency>`. If the invoice's currency differs from the
  company's currency, the currency exchange rate is automatically displayed.

In the :guilabel:`Invoice Lines` tab:

- :guilabel:`Product`: Click :guilabel:`Add a line`, then search for and select the product.
- :guilabel:`Quantity`
- :guilabel:`Price`
- :doc:`Taxes <taxes>` (if applicable)

> To access the product catalog and view all items in an organized display, click :guilabel:`Catalog`.

Author: @auva-odoo Yes, you're right, I didn't know it was already available ;) Thanks!

To select products, click :icon:`fa-shopping-cart` :guilabel:`Add` on the product card. The quantity
can be adjusted using the :icon:`fa-minus-square` :guilabel:`(minus-square)` or
:icon:`fa-plus-square` :guilabel:`(plus-square)` buttons. :guilabel:`Remove` products if needed.
Then, click :guilabel:`Back to Invoice` to return to the invoice. The selected catalog items will
appear in the invoice lines.

.. tip::
   To display the total amount of the invoice in words, go to :menuselection:`Accounting -->
   Configuration --> Settings` and activate the :guilabel:`Total amount of invoice in letters`
   option.

The :guilabel:`Journal Items` tab displays the accounting entries created. Additional invoice
information such as the :guilabel:`Customer Reference`, :guilabel:`Payment Reference`, :doc:`Fiscal
Positions <taxes/fiscal_positions>`, :doc:`Incoterms <customer_invoices/incoterms>`, and more can be
added or modified in the :guilabel:`Other Info` tab.

.. note::

>    Odoo initially creates invoices in :guilabel:`Draft` status. Draft invoices have no accounting

Reviewer: I cannot comment on this line because you didn't change it in this PR, but could we use "invoice" instead of "document"  in the Invoice confirmation section?

To send the invoice to the customer:

#. Click :guilabel:`Print & Send`.
#. If a :ref:`default invoice layout <studio/pdf-reports/default-layout>` hasn't been customized, a
   :guilabel:`Configure your document layout` pop-up window will appear. Configure the invoice
   layout and click :guilabel:`Continue`.

   .. note::
      To add a QR code for banking app payments to the invoice, enable the :guilabel:`QR Code`
      option in the :guilabel:`Configure Your Document Layout` window.

#. In the :guilabel:`Print & Send` window:

   - If a preferred :guilabel:`Invoice sending` method was set in the contact form, it will be
     selected by default. However, a different sending option can be chosen if needed.
   - If no preferred :guilabel:`Invoice sending` method was set in the contact form, select how to
     send this invoice to the customer.

#. Click :guilabel:`Print & Send` if the :guilabel:`by Email` option is selected, or click
   :guilabel:`Print`.

To send and print multiple invoices, go to :menuselection:`Accounting --> Customers --> Invoices`
and select them in the :guilabel:`Invoices` list view. Then click the :icon:`fa-cog`

> :guilabel:`Actions` menu and select :guilabel:`Print & Send`. The :guilabel:`Print & Send` window

Author: Good catch, thanks!

Invoice sending
===============

To set a preferred :guilabel:`Invoice sending` method for a customer, go to
:menuselection:`Accounting --> Customers --> Customers` and select the customer. In the
:guilabel:`Accounting` tab of the contact form, select the preferred :guilabel:`Invoice sending`
method in the :guilabel:`Customer Invoices` section.

To send the invoice to the customer, navigate back to the invoice record and follow these steps:

#. Click :guilabel:`Print & Send`.
#. If a :ref:`default invoice layout <studio/pdf-reports/default-layout>` hasn't been customized, a

>    :guilabel:`Configure your document layout` pop-up window will appear. Configure the invoice

Reviewer: ```suggestion
   :guilabel:`Configure your document layout` pop-up window appears. Configure the invoice
```

   layout and click :guilabel:`Continue`.

   .. note::
      - The document layout can be changed at any time in the general settings.
      - To add a QR code for banking app payments to the invoice, enable the :guilabel:`QR Code`

>         option in the :guilabel:`Configure Your Document Layout` window. This option can always be
>         changed by going to :menuselection:`Accounting --> Configuration --> Settings`, and changing
>         the :guilabel:`QR Codes` option in the :guilabel:`Customer Payments` section.

Reviewer: ```suggestion
        option in the :guilabel:`Configure Your Document Layout` window. To modify this option, go to :menuselection:`Accounting --> Configuration --> Settings`, scroll down to the :guilabel:`Customer Payments` section, and enable/disable the :guilabel:`QR Codes` option.
```
- to get rid of "always"
- I think we can use the passive voice here

#. In the :guilabel:`Print & Send` window:

>    - If a preferred :guilabel:`Invoice sending` method was set in the contact form, it will be
>      selected by default. However, a different sending option can be chosen if needed.

Reviewer: ```suggestion
   - If a preferred :guilabel:`Invoice sending` method was set in the contact form, it is
     selected by default. A different sending option can be chosen if needed.
```

>    - If no preferred :guilabel:`Invoice sending` method was set in the contact form, select how to
>      send this invoice to the customer.

Reviewer: ```suggestion
   - If no preferred :guilabel:`Invoice sending` method was set in the contact form, select the method to use for sending the invoice to the customer.
```

#. Click :guilabel:`Print & Send` if the :guilabel:`by Email` option is selected, or click
   :guilabel:`Print`.

To send and print multiple invoices, go to :menuselection:`Accounting --> Customers --> Invoices`,
select them in the :guilabel:`Invoices` list view and click :guilabel:`Print & Send`. The
:guilabel:`Print & Send` window displays the selected invoice sending methods based on the preferred
method set.

> A banner will appear on the selected invoices to indicate they are part of

Reviewer: ```suggestion
A banner is added to the selected invoices to indicate they are part of
```
or appears on (the aim is to get rid of the future tense here)
